aptos-move/aptos-vm/src/block_executor/mod.rs

```diff
@@ -114,7 +115,7 @@ impl BlockExecutorTransactionOutput for AptosTransactionOutput {
 
     /// Should never be called after incorporate_delta_writes, as it
     /// will consume vm_output to prepare an output with deltas.
-    fn aggregator_v1_write_set(&self) -> HashMap<StateKey, WriteOp> {
+    fn aggregator_v1_write_set(&self) -> HashMap<AggregatorID, WriteOp> {
```

Comment:
ditto I like StateKeys here :) 
but maybe this requires having 2 separate associated types in AggregatorResolver.